source/reference/minio-server/settings/core.rst

```diff
@@ -0,0 +1,331 @@
+      Use :mc:`mc alias set` to configure an ALIAS.
+
+Add temporary credentials to use with :mc:`mc alias` commands.
+For example, to use for scripting.
+
+Examples
+++++++++
+
+**Static Credentials**
+
+.. tab-set::
+
+   .. tab-item:: Syntax
+
+      .. code-block:: shell
+         :class: copyable
+      
+         export MC_HOST_<alias>=https://<Access Key>:<Secret Key>@<YOUR-S3-ENDPOINT>
+
+   .. tab-item:: Example
+
+      .. code-block:: shell
+         :class: copyable
+
+         export MC_HOST_myalias=https://Q3AM3UQ867SPQQA43P2F:zuf+tfteSlswRu7BJ86wekitnifILbZam1KYY3TG@play.min.io
+
+
+**Rotating Credentials**
+
+.. tab-set::
+
+   .. tab-item:: Syntax
+
+      .. code-block:: shell
+         :class: copyable
+         
+         export MC_HOST_<alias>=https://<Access Key>:<Secret Key>:<Session Token>@<YOUR-S3-ENDPOINT>
+
+   .. tab-item:: Example
+
+      .. code-block:: shell
+         :class: copyable
+
+         export MC_HOST_myalias=https://Q3AM3UQ867SPQQA43P2F:zuf+tfteSlswRu7BJ86wekitnifILbZam1KYY3TG:eyJhbGciOiJIUzUxMiIsInR5cCI6IkpXVCJ9.eyJhY2Nlc3NLZXkiOiJOVUlCT1JaWVRWMkhHMkJNUlNYUiIsImF1ZCI6IlBvRWdYUDZ1Vk80NUlzRU5SbmdEWGo1QXU1WWEiLCJhenAiOiJQb0VnWFA2dVZPNDVJc0VOUm5nRFhqNUF1NVlhIiwiZXhwIjoxNTM0ODk2NjI5LCJpYXQiOjE1MzQ4OTMwMjksImlzcyI6Imh0dHBzOi8vbG9jYWxob3N0Ojk0NDMvb2F1dGgyL3Rva2VuIiwianRpIjoiNjY2OTZjZTctN2U1Ny00ZjU5LWI0MWQtM2E1YTMzZGZiNjA4In0.eJONnVaSVHypiXKEARSMnSKgr-2mlC2Sr4fEGJitLcJF_at3LeNdTHv0_oHsv6ZZA3zueVGgFlVXMlREgr9LXA@play.min.io
+
+Batch Replication
+-----------------
+
+.. tab-set::
+
+   .. tab-item:: Environment Variable
+
+      .. envvar:: MINIO_BATCH_REPLICATION_WORKERS
+
+         *Optional*
+
+         Enable parallel workers by specifying the maximum number of processes to use when performing the batch application job.
+
+   .. tab-item:: Configuration Setting
+
+      This setting does not have a configuration variable setting.
+      Use the Environment Variable instead.
+
+Data Compression
+----------------
+
+The following section documents settings for enabling data compression for objects.
+See :ref:`minio-data-compression` for tutorials on using these configuration settings.
+
+Allow Encryption
+~~~~~~~~~~~~~~~~
+
+.. tab-set::
+
+   .. tab-item:: Environment Variable
+      :sync: envvar
+
+      .. envvar:: MINIO_COMPRESSION_ALLOW_ENCRYPTION
+
+   .. tab-item:: Configuration Setting
+      :sync: config
+  
+      .. mc-conf:: compression allow_encryption
+         :delimiter: " "
+
+*Optional*
+
+Set to ``on`` to encrypt objects after compressing them.
+Defaults to ``off``.
+
+.. admonition:: Encrypting compressed objects may compromise security
+   :class: warning
+
+   MinIO strongly recommends against encrypting compressed objects.
+   If you require encryption, carefully evaluate the risk of potentially leaking information about the contents of encrypted objects.
+
+Enable Compression
+~~~~~~~~~~~~~~~~~~
+
+.. tab-set::
+
+   .. tab-item:: Environment Variable
+      :sync: envvar
```

Comment:
No, I think this just impacts how the page displays when someone clicks on Configuration Setting on one, it changes all of the ones on the page over the config settings.

Shouldn't affect anything else with how links work to the envvar or conf settings otherwise.